mitosheet/src/mito/components/toolbar/FormulaTabContents.tsx

```diff
@@ -59,6 +59,7 @@ export const FormulaTabContents = (
                                 ...props.editorState,
                                 formula: `=${functionObject.function}(${currentFormula.startsWith('=') ? currentFormula.substring(1) : currentFormula}`,
                             })
+                            document.getElementById('cell-editor-input')?.focus();
```

Comment:
document is the wrong thing to use here, it may lead to bugs when there are multiple mitosheets in the same page.

Reply:
Ah good point - done